source/ios/AdaptiveCards/ADCIOSFramework/ADCIOSFramework/ACRContentStackView.mm

```diff
@@ -0,0 +1,111 @@
+//
+//  ACRContentStackView
+//  ACRContentStackView.mm
+//
+//  Copyright © 2017 Microsoft. All rights reserved.
+//
+
+#include "ACRContentStackView.h"
+
+@implementation ACRContentStackView
+
+-(instancetype)init 
+{
+    self = [super init];
+
+    if(self) 
+    {
+        _stackView = [[UIStackView alloc] init];
+        [self config];
+    } 
+
+    return self;
+}
+
+-(instancetype)initContentViewWithFrame:(CGRect) frame
```

Comment:
The frame of the stack view is the same as the frame of the ACRContentStackView, right? initWithFrame is the expected name for this initializer.

In any case, every UIView is expected implement initWithFrame, or else use the superclass's implementation (which won't work here).

---
In reply to: [128333362](https://github.com/Microsoft/AdaptiveCards/pull/488#discussion_r128333362) [](ancestors = 128333362,128110303)